opentracing/span.py

```diff
@@ -203,3 +205,11 @@ def __exit__(self, exc_type, exc_val, exc_tb):
                                                 'val': exc_val,
                                                 'tb': exc_tb})
         self.finish()
+
+    def log_event(self, event, payload=None):
+        """DEPRECATED"""
+        return self
+
+    def log(self, **kwargs):
```

Comment:
kind of a shame to kill this one. It's nearly equivalent to `log_kv` except requires "safe" field labels, which incidentally is all you have in the examples for `log_kv`